### PR TITLE
Adding non-kuryr content

### PR DESCRIPTION
**Pull request by dpaterson**
- Overview
- DCN description
- Networking
 - Spine & Leaf
 - Impairment
- Additional drawings
- Added Mark's impairment and caching demos to slides
- Tried to add transition from DCN to Kuryr etc..
- Added closing and qa.

TODO: Need a better summary slide(s)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -327,8 +600,6 @@
 
 ## Thank you!
 
-**Slides can be found at:**
-
 **Daniel Mellado**
 
 ![email](assets/email.png)   [dmellado@redhat.com](mailto:dmellado@redhat.com)
```
Comment (danielmellado) [inline index.html]: Hi David, overall it all looks really nice. I think you'd like to add yourselves here as well.

Thanks!

**Review by danielmellado — COMMENTED** [review]
Hi David,

Thanks for the update, I've added just one nit there.

File: index.html (hunk 1 above)
Comment (dpaterson) [reply index.html]: Sorry Daniel did you want me to put **Slides can be found at:** back in? If so did you want me to also add the link to your repo?

Comment (danielmellado) [reply index.html]: Heh, nope, don't worry about that, I'll add that link just after I upload these to my GhH pages. I was just thinking about you adding yourselves after that, just like I have my name and mail and so

Comment (dpaterson) [reply index.html]: The latest push to this PR has our names, email and IRC handle.

**Pull request merged**